src/Tool/Commands/SqsCommand.cs

```diff
@@ -64,49 +61,33 @@ public SqsCommand(SharedOptions shared, string profile, string regionName, strin
         }
 
         this.prefix = prefix;
+        aws = new AwsQuery();
     }
 
-    string prefix;
+    readonly AwsQuery aws;
+    readonly string prefix;
     int metricsReceived;
     List<string> queueNames;
     string[] ignoredQueueNames;
 
     protected override async Task<QueueDetails> GetData(CancellationToken cancellationToken = default)
     {
-        var endTime = DateTime.UtcNow.Date.AddDays(1);
-        var startTime = endTime.AddDays(-30);
-
-        var cloudWatch = new AmazonCloudWatchClient();
-        Out.WriteLine($"Loading CloudWatch metrics from {cloudWatch.Config.RegionEndpoint.SystemName}.");
+        var aws = new AwsQuery();
+        Out.WriteLine($"Loading CloudWatch metrics from {aws.CloudWatchRegion}.");
 
         var data = new ConcurrentBag<QueueThroughput>();
 
         var tasks = queueNames.Select(async queueName =>
         {
-            var req = new GetMetricStatisticsRequest
-            {
-                Namespace = "AWS/SQS",
-                MetricName = "NumberOfMessagesDeleted",
-                StartTimeUtc = startTime,
-                EndTimeUtc = endTime,
-                Period = 86400, // 1 day
-                Statistics = new List<string> { "Sum" },
-                Dimensions = new List<Dimension> {
-                    new Dimension { Name = "QueueName", Value = queueName }
-                }
-            };
-
-            var resp = await cloudWatch.GetMetricStatisticsAsync(req, cancellationToken);
-
-            var maxThroughput = resp.Datapoints.OrderByDescending(d => d.Sum).FirstOrDefault()?.Sum ?? 0;
+            var maxThroughput = await aws.GetMaxThroughput(queueName, cancellationToken).ConfigureAwait(false);
 
             data.Add(new QueueThroughput
             {
                 QueueName = queueName,
-                Throughput = (int)maxThroughput
+                Throughput = maxThroughput
             });
 
-            Interlocked.Increment(ref metricsReceived);
+            _ = Interlocked.Increment(ref metricsReceived);
```

Comment:
Because the analyzer [IDE0058](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0058). The method returns the incremented value.